data/data/powervs/loadbalancer/versions.tf

```diff
@@ -0,0 +1,28 @@
+################################################################
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Licensed Materials - Property of IBM
+#
+# ©Copyright IBM Corp. 2021
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################
+
+terraform {
+  required_providers {
+    ibm = {
+      source  = "ibm-cloud/ibm"
+      version = "1.24.0"
+    }
+  }
+}
```

Comment:
Can we have one global provider instead of having it in every folder?

Reply:
Yes

Reply:
@mkumatag we can't have one global provider. It fails with this
```
│ Error: Failed to query available provider packages
│
│ Could not retrieve the list of available versions for provider hashicorp/ibm: provider registry registry.terraform.io does not have a provider named registry.terraform.io/hashicorp/ibm
│
│ Did you intend to use ibm-cloud/ibm? If so, you must specify that source address in each module which requires that provider. To see which modules are currently depending on
│ hashicorp/ibm, run the following command:
│     terraform providers
```

Reply:
iiuc we won't have to carry any of these provider files forward, and can remove them when we're relying on the installer to set up the tf env.